src/pages/Portfolio/ImpactDashboard/LendingInsights.vue

```diff
@@ -105,104 +105,215 @@
 				</p>
 			</div>
 		</div>
-		<div class="tw-flex tw-items-center tw-justify-between">
-			<h2 class="tw-text-h3 tw-mb-3 md:tw-mb-2 tw-text-eco-green-4 tw-text-center md:tw-text-left">
-				Lending insights
-			</h2>
-			<div class="tw-flex tw-items-center tw-gap-2">
-				<!-- To-Do: Functionality work regarding YTD and lifetime tab column (MP-1858) -->
-				<h2
-					class="tw-text-h3 tw-mb-3 md:tw-mb-2 tw-text-eco-green-4
-					tw-text-center md:tw-text-left hover:tw-underline hover:tw-decoration-2"
-				>
-					2025
-				</h2>
-				<h2
-					class="tw-text-h3 tw-mb-3 md:tw-mb-2 tw-text-eco-green-4
-					tw-text-center md:tw-text-left hover:tw-underline hover:tw-decoration-2"
-				>
-					Lifetime
-				</h2>
-			</div>
+		<div>
+			<kv-tabs @tab-changed="setActiveTab">
+				<template #tabNav>
+					<h2 class="tw-text-h3 tw-mb-3 md:tw-mb-2 tw-text-eco-green-4 tw-text-center md:tw-text-left">
+						Lending insights
+					</h2>
+					<kv-tab
+						for-panel="ytd"
+						class="tab-header"
+						v-kv-track-event="['portfolio', 'click', 'stats-YTD']"
+					>
+						{{ yearToDate }}
+					</kv-tab>
+					<kv-tab
+						for-panel="lifetime"
+						class="tab-header"
+						v-kv-track-event="['portfolio', 'click', 'stats-Lifetime']"
+					>
+						Lifetime
+					</kv-tab>
+				</template>
+				<template #tabPanels>
+					<kv-tab-panel id="ytd">
+						<!-- Total amount lent -->
+						<kv-grid as="dl" class="stats-container-exp">
+							<div class="tw-col-span-12 md:tw-col-span-6 lg:tw-col-span-3">
+								<kv-loading-placeholder
+									v-if="loading"
+									class="stat-placeholder"
+									style="width: 7rem;"
+								/>
+								<dt v-show="!loading" class="stat-value">
+									{{ currentYearAmountLent }}
+								</dt>
+								<dd class="stat-def">
+									Total amount lent in {{ yearToDate }}
+								</dd>
+								<router-link
+									class="stat-link"
+									to="/portfolio/loans"
+									v-kv-track-event="['portfolio', 'click', 'total-amount-lent-details']"
+								>
+									My loans
+									<kv-material-icon
+										class="tw-ml-0.5 tw-w-2 tw-h-2"
+										:icon="mdiArrowRight"
+									/>
+								</router-link>
+							</div>
+							<!-- Lending percentile -->
+							<div
+								class="tw-col-span-12 md:tw-col-span-6 lg:tw-col-span-3
+										tw-bg-eco-green-3 tw-rounded"
+							>
+								<kv-loading-placeholder
+									v-if="loading"
+									class="stat-placeholder"
+									style="width: 7rem;"
+								/>
+								<dt v-show="!loading" class="stat-value">
+									{{ currentYearPercentile }}
```

Comment:
I'm seeing the same number for my user. Are we using the correct number to display?

<img width="399" height="254" alt="image" src="https://github.com/user-attachments/assets/b8e10bc3-4df3-47ab-ab2a-3a370b066296" />